packages/openapi-to-graphql/src/resolver_builder.ts

```diff
@@ -1225,45 +1226,37 @@ function resolveRuntimeExpression(
       return result
 
       // CASE: parameter in body
-    } else if (value.startsWith('$response.body#')) {
-      const tokens = JSONPath({
-        path: value.split('body#/')[1],
-        json: root
-      })
-      if (Array.isArray(tokens) && tokens.length > 0) {
-        return tokens[0]
-      } else {
-        httpLog(`Warning: could not extract parameter '${paramName}' from link`)
-      }
+    } else if (runtimeExpression.startsWith('$response.body#')) {
+      return JSONPointer.get(root, runtimeExpression.split('body#')[1])
```

Comment:
Switch from JSONPath to JSONPointer. It did not make sense to use JSONPath here. It just happened to work for simple parameters.